android/source/VideoLocker/res/values-en/strings.xml

```diff
@@ -22,8 +22,6 @@
     <string name="login_title">Sign in to edX</string>
 
     <!-- Navigation Drawer Labels -->
-    <string name="name_default">Morgank</string>
-    <string name="email_default">morgan.knuston@yahoo.com</string>
```

Comment:
I'll make that change. Although the defaults for the TextViews should never be used, I just realized it's still useful when viewing the layout on Studio.